tests/phpunit/unit/Configuration/ResourceManagerTest.php

```diff
@@ -178,9 +178,9 @@ public function testDefaultRequest()
         $app = new Application(['resources' => $config]);
         $this->assertEquals('cli', $config->getRequest('protocol'));
         $this->assertEquals('bolt.dev', $config->getRequest('hostname'));
-        $this->assertEquals('http://bolt.dev/bolt', $config->getUrl('canonical'));
+        $this->assertEquals('cli://bolt.dev/bolt', $config->getUrl('canonical'));
         $this->assertEquals('cli://bolt.dev', $config->getUrl('host'));
-        $this->assertEquals('http://bolt.dev/', $config->getUrl('rooturl'));
+        $this->assertEquals('cli://bolt.dev/', $config->getUrl('rooturl'));
```

Comment:
@CarsonF: Yeah, my thing is basically just getting the tests to behave predictably and understanding what I misunderstood, I won't be putting that much time into it since it'll be removed soon anyway.